falcon/response.py

```diff
@@ -132,6 +132,8 @@ class Response(object):
                 the Response instance itself (self).
 
         options (dict): Set of global options passed from the API handler.
+
+        headers (list): Read-only list of all headers set for Response
```

Comment:
Documentation sentences for other members of the class end with a full stop (period), let us add one here too.